### PR TITLE
Add Mina for deploy

### PR DESCRIPTION
**Pull request by enricostano**
Adding production deploy stuff using [Mina](https://github.com/mina-deploy/mina).

`bundle exec mina deploy` and you're ready to go!

### CODE REVIEW

**Comment by andresgutgon** [comment]
:+1:

File: config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+require 'mina/bundler'
+require 'mina/rails'
+require 'mina/git'
+
+set :domain, '10.0.3.80'
```
Comment (sauloperez) [inline config/deploy.rb]: We'll have to add something like Foreman?

Comment (enricostano) [inline config/deploy.rb]: Not sure if Foreman, but we'll need to parametrize this for sure

Comment (sauloperez) [inline config/deploy.rb]: There must be some solution based on yml or something like that, for sure.

**Pull request merged**